### ISSUE TITLE
Webserver stops working and log is filled with errors after a couple of seconds.

### ISSUE THREAD

**Issue by stekern** [favll/pogom#138]
After a couple of accounts are logged in successfully, I start getting heaps of these messages and then the webserver stops working. 

```
2016-08-09 07:27:50,710 [   auth_ptc] [  ERROR] PTC Login failed.
2016-08-09 07:27:50,732 [     pgoapi] [  ERROR] Error in worker thread. Returning empty response. Error: local variable 'r' referenced before assignment
2016-08-09 07:27:50,735 [       scan] [   INFO] Map Download failed. Trying again.
```

and these

```
2016-08-09 07:30:36,621 [     pgoapi] [   INFO] Server seems to be busy or offline!
2016-08-09 07:30:36,623 [     pgoapi] [   INFO] Server seems to be busy or offline!
2016-08-09 07:30:36,626 [     pgoapi] [   INFO] Server seems to be busy or offline!
2016-08-09 07:30:36,629 [     pgoapi] [   INFO] Server seems to be busy or offline!
```

**Comment by favll** [comment]
Where are you hosting your clone?

**Comment by stekern** [comment]
@favll I'm hosting it locally on my Macbook Pro running OS X.

**Comment by TravisDean** [comment]
I'm getting the same error messages from the first box. It was working perfectly fine, then I stopped the server and stepped away for about an hour yesterday. When I came back, continual PTC login failures. I've tried making new accounts that I can login successfully with on the website, restarting the VM, repulling the repo, and even switching ISPs (via testing with tethering). All hosted locally.

I think it might be a PGoApi issue though, since the exact same issue occurs when I try to log in just using pgoapi.
